pkg/cmd/unpack/cmd.go

```diff
@@ -41,27 +41,42 @@ filesystem. By default, it attempts to find the modelkit in local storage; if
 not found, it searches the remote registry and retrieves it. This process
 ensures that the necessary components are always available for unpacking,
 optimizing for efficiency by fetching only specified components from the
-remote registry when necessary`
+remote registry when necessary.
+
+The content that is unpacked can be limited via the --filter (-f) flag. For example,
+use
+    --filter=model
+to unpack only the model, or 
+    --filter=datasets:my-dataset
+to unpack only the dataset named 'my-dataset'. Filtering for specific code/datasets/docs
+will match on either name or the path of the field.`
 
 	example = `# Unpack all components of a modelkit to the current directory
 kit unpack myrepo/my-model:latest -d /path/to/unpacked
 
 # Unpack only the model and datasets of a modelkit to a specified directory
```

Comment:
Can we have an example that shows multiple filters. Also a better explanation for the cases  where multiple filter contents can be used.

Reply:
Hm, it is getting long and complicated. Maybe it makes sense to have a separate "Filtering" section in the help page. I'll come up with something.

Reply:
I couldn't add a section (the whole desc is indented so it looks weird) but I've expanded this description and added more examples.